src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ShowPlanGraph/Description.cs

```diff
@@ -44,45 +47,69 @@ public bool IsClusteredMode
                 this.isClusteredMode = value;
             }
         }
-
-        public bool HasMissingIndex
+        
+        public List<MissingIndex> MissingIndices
         {
-            get { return this.hasMissingIndex; }
+            get { return this.missingIndexes; }
+            set
+            {
+                this.missingIndexes = value;
+            }
         }
 
+        #endregion
+
+        #region Member variables
+
+        private string title = string.Empty;
+        private string queryText = string.Empty;
+        private string toolTipQueryText = string.Empty;
+        private string clusteredMode = string.Empty;
+        private bool isClusteredMode = false;
+        private List<MissingIndex> missingIndexes = new List<MissingIndex>();
+        #endregion
+    }
+
+    public class MissingIndex
+    {
+        public string MissingIndexCaption
+        {
+            get { return this.missingIndexCaption; }
+            set
+            {
+                this.missingIndexCaption = value;
+            }
+        }
         public string MissingIndexQueryText
         {
             get { return this.missingIndexQueryText; }
+            set
+            {
+                this.missingIndexQueryText = value;
+            }
         }
 
         public string MissingIndexImpact
         {
             get { return this.missingIndexImpact; }
+            set
+            {
+                this.missingIndexImpact = value;
```

Comment:
For these you can just remove the implementations and use the auto setter-getter functionality when there isn't any additional logic you're doing